insights/parsers/httpd_conf.py

```diff
@@ -104,11 +104,13 @@ class HttpdConf(LegacyItemAccess, Parser):
 
     Attributes:
         data (dict): Dictionary of parsed data in format of {option: value} or
-                     {section: {option: value}}.
-        full_data (dict): Dictionary of parsed data with key being option and value a named tuple
-                          with the following properties:
+                     {section: {option: value}}. Stores last found value.
+        full_data (dict): Dictionary of parsed data with key being option and value a list of named
```

Comment:
...with key being *the* option...

Without *the*, I couldn't parse the sentence for a half a minute.